### ISSUE TITLE
`color-function-notation` show error for scss `rgba` function

### ISSUE THREAD

**Issue by rgant** [stylelint-scss/stylelint-config-standard-scss#11]
Copy of issue from @qmhc stylelint-scss/stylelint-scss#563

> Will take the `color-function-notation` error when using scss rgba function like:
> 
> ```css-scss
> .a {
>   color: rgba($base-color, 0.8);
> }
> ```
> 
> A config `'color-function-notation': 'modern'` extends from `stylelint-config-standard` (using `stylelint-config-standard-scss`), it cannot distinguish rgba between css and scss.

I am also facing this issue, which I have avoided by adding to my .stylelintrc.yml
```
 extends:
   - stylelint-config-standard-scss
 rules:
+  # Modern color function notation isn't compatible with SCSS
+  color-function-notation: null
```

**Comment by lachieh** [comment]
Once stylelint/stylelint#5671 is closed, this change will not be required to prevent unnecessary errors.

**Comment by jeddy3** [comment]
Fixed upsteam.
